lib/Element.js

```diff
@@ -181,23 +164,63 @@ function Element(document, $elm) {
     }
   });
 
+  function radioButtonChecked(value) {
+    const oldValue = (getAttribute("checked") === "checked");
```

Comment:
checked is a boolean. so comparision should be `typeof getAttribute("checked") !== "undefined"`